src/lighteval/models/transformers/transformers_model.py

```diff
@@ -221,11 +235,21 @@ def get_model_sha(self):
         return _get_model_sha(repo_id=self.pretrained, revision=self.revision)
 
 
-class BaseModel(LightevalModel):
+@dataclass
+class BaseModelConfig(TransformersModelConfig):
+    def __post_init__(self):
+        super()
```

Comment:
I assume you meant to call the parent's `__post_init__` method:
```suggestion
        super()__post_init__()
```